execute_data_volume.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+mkdir data_volume
+
+for i in `seq 600000 6000000 59986050`;
+do
+	export LINEITEMS=$i
```

Comment:
Wouldn't it make more sense to just use a parameter for the script instead of a ENVVAR?